docs/project/documentation.md

```diff
@@ -79,6 +79,55 @@
+
+As a replacement we found several other tools. Most of them were sadly even less convenient or just overly complicated compared to docusaurus. One of the top candidates was [Sphinx](https://www.sphinx-doc.org/en/master/), but it uses a very odd syntax over the regular Markdown everyone in the team is used to.
+
+When looking for a potential host for it we found [Read the Docs](https://about.readthedocs.com/?ref=readthedocs.com). It can even fully automate previews and deployments from a GitHub repository by simply setting up a Webhook in it. This feature spared us the creation of a dedicated workflow and the debugging of it, which took a good while with Docusaurus originally. So this is the host we went with for the documentation, instead of GitHub-Pages.
+
+While skimming through the documentation of the new host, we found out that it also works with a Markdown based framework called [MkDocs](https://www.mkdocs.org/). After reading up a bit on MkDoc's customization possibilities we found the [Material Theme for MkDocs by Squidfunk](https://squidfunk.github.io/mkdocs-material/) to be suitable for our needs. It provides more or less the same features as Docusaurus, just without the complications we had. As so, our final decision regarding the documentation side of things was to use MkDocs with the Material Theme and host it on Read the Docs.
+
+### Linting and Linters
+
+When it comes to linting C#, dotnet has a default formatter that can also enforce some style rules and patterns. The range of options one can choose from is not very wide, however. To have more freedom in terms of details and choice, we decided to integrate [stylecop](https://marketplace.visualstudio.com/items?itemName=ChrisDahlberg.StyleCop), a static code analyzer for C#, into our project.
+
+### REST API Framework
+
+So we won't have to create our entire REST API from scratch, we decided to use a framework that is part of the dotnet ecosystem. We will be using an [ASP.NET Core Web API](https://dotnet.microsoft.com/en-us/apps/aspnet/apis) te get things going.
+
+Another reason for our choice was the compatibility and easy integration with the Entity Framework
+
+### Testing REST Endpoints
+
+To test and develop the endpoints of the REST API we will be using a workspace on [Postman](https://www.postman.com/). It is a tool that supports automated testing of API endpoints with lots of utility. The reason we went with this option is that all of us worked with it several times before.
+
+### Handling SQL Transaction Errors
+
+Considering the fact that our app will be booking digital money, we want rule out any possibility of anomalies that can alter the total amount of money in the system. Especially since the our app allows multiple users to use it at the same time.
```

Comment:
"Especially since the our app ..." (last sentence) -> either remove "the" or "our"